MAINTAINERS.md

```diff
@@ -0,0 +1,87 @@
+# Deis Maintainers
+
+This document serves to describe the leadership structure of the Deis project, and list the current
+project maintainers.
+
+# What is a maintainer?
+
+(Unabashedly stolen from the [Docker](https://github.com/docker/docker/blob/master/MAINTAINERS) project)
+
+There are different types of maintainers, with different responsibilities, but
+all maintainers have 3 things in common:
+
+1. They share responsibility in the project's success.
+2. They have made a long-term, recurring time investment to improve the project.
+3. They spend that time doing whatever needs to be done, not necessarily what
+is the most interesting or fun.
+
+Maintainers are often under-appreciated, because their work is harder to appreciate.
+It's easy to appreciate a really cool and technically advanced feature. It's harder
+to appreciate the absence of bugs, the slow but steady improvement in stability,
+or the reliability of a release process. But those things distinguish a good
+project from a great one.
+
+# Deis maintainers
+
+Deis has two groups of maintainers in addition to our beloved Benevolent Dictator for Life.
+
+## BDFL
+
+Deis follows the timeless, highly efficient and totally unfair system known as [Benevolent dictator
+for life](http://en.wikipedia.org/wiki/Benevolent_Dictator_for_Life). Gabriel Monroy (@gabrtv), as
```

Comment:
handles like @gabrtv aren't being properly handled in markdown. Maybe we should use direct links instead? Something like [Gabriel Monroy](https://github.com/gabrtv)